client/pseudonymsys_ca_ec.go

```diff
@@ -15,8 +15,8 @@ type PseudonymsysCAClientEC struct {
 	prover *dlogproofs.SchnorrECProver
 }
 
-func NewPseudonymsysCAClientEC(endpoint string) (*PseudonymsysCAClientEC, error) {
-	genericClient, err := newGenericClient(endpoint)
+func NewPseudonymsysCAClientEC(conn *grpc.ClientConn) (*PseudonymsysCAClientEC, error) {
```

Comment:
It would be best to enforce `<whatever>Client` naming convention. Here I see you use:
```
PseudonymsysCAClientEC
```
but I would expect something like:
```
PseudonymsysCaEcClient
```
Please consider unifying this in one of the following PRs (not in this one).